docs/source/en/model_doc/openai-gpt.mdx

```diff
@@ -57,6 +57,27 @@ python -m spacy download en
 If you don't install `ftfy` and `SpaCy`, the [`OpenAIGPTTokenizer`] will default to tokenize
 using BERT's `BasicTokenizer` followed by Byte-Pair Encoding (which should be fine for most usage, don't worry).
 
+## Resources
+
+A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with DistilBERT. If you're interested in submitting a resource to be included here, please feel free to open a Pull Request and we'll review it! The resource should ideally demonstrate something new instead of duplicating an existing resource.
+
+<PipelineTag pipeline="text-classification"/>
+
+- A blog post on [outperforming OpenAI GPT-3 with SetFit for text-classification](https://www.philschmid.de/getting-started-setfit).
+
+<PipelineTag pipeline="text-generation"/>
+
+- OpenAI GPT is supported by this [example script](https://github.com/huggingface/transformers/blob/main/examples/pytorch/text-generation/run_generation.py) and [notebook](https://colab.research.google.com/github/huggingface/blog/blob/main/notebooks/02_how_to_generate.ipynb).
+
+<PipelineTag pipeline="token-classification"/>
+
+- A course material on [Byte-Pair Encoding tokenization](https://huggingface.co/course/en/chapter6/5).
+- A course material on [training a causal language model from scratch](https://huggingface.co/course/en/chapter7/6).
+
+<PipelineTag pipeline="causal-language-modeling"/>
```

Comment:
We can delete this section now since we added causal language modeling under text generation.